src/components/Orders.css

```diff
@@ -0,0 +1,7 @@
+.Orders {
```

Comment:
in the future we should avoid having components that have a fixed width or height because we want things to be responsive (change size based on the screen size). This will be updated this week so ok to update for now.